dolphin/coordination.py

```diff
@@ -30,7 +30,10 @@
 coordination_opts = [
     cfg.StrOpt('backend_url',
                default='redis://127.0.0.1:6379',
-               help='The back end URL to use for distributed coordination.')
+               help='The back end URL to use for distributed coordination.'),
+    cfg.IntOpt('expiration',
+               default=100,
```

Comment:
Can this expiration time come from some config file? This will make it tunable

Reply:
Yes, we only give it a default value to this config.
We can use
```
[coordination]
expiration = xxx
```
in dolphin.conf to modify it.
I'll add this config sample to release note of this pr

Reply:
ok